students/morgan/mailprog/mailprog/ui.py

```diff
@@ -0,0 +1,33 @@
+from mailprog.functions import list_donors, donation_input, print_report, email_all
```

Comment:
this is where OO is kinda nice --  you'd only need to import the name of the class